samples/maps/series-tiledwebmap/mappoint-on-twm/demo.js

```diff
@@ -0,0 +1,60 @@
+Highcharts.mapChart('container', {
+    title: {
+        text: 'Highcharts Maps TiledWebMap Series with map points'
+    },
+
+    mapNavigation: {
+        enabled: true
+    },
+
+    mapView: {
+        projection: {
+            name: 'WebMercator'
+        },
```

Comment:
Isn't this obsolete? The TWM series is now able to force the projection.

Reply:
I left it as an indication for the users, to tell them which projection is being used to display the map points. But of course, at the current state of the TWM it is obsolete.